modules/auxiliary/scanner/http/docker_enum.rb

```diff
@@ -0,0 +1,49 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Auxiliary::Scanner
+  include Msf::Auxiliary::Report
```

Comment:
I don't think you're using this yet?

Reply:
You are right!
I used another module as a template.

I could delete the include or use the report functionality.
What do you think is best?

Reply:
It's all right. Most of us start with another module, but not all modules are good examples! You can remove it for now or use one of the `report_*` methods from it!

Reply:
Preference is on reporting if you can. :)

Reply:
In 00f4d3967c I add some basic reporting functionality.

I could also add some reporting for:
OS, Arch, KernelVersion

I think that could be quite useful for exploitation, anyhow I'm not sure if it's ideal that this module add that information about the remote host.

Ideas? Suggestions?

Reply:
Addressed in 7d8b9a90d7